tests/core/mempool/test_mempool_manager.py

```diff
@@ -552,3 +574,45 @@ async def test_get_items_not_in_filter() -> None:
     sb2_and_3_filter = PyBIP158([bytearray(sb2_name), bytearray(sb3_name)])
     result = await mempool_manager.get_items_not_in_filter(sb2_and_3_filter)
     assert result == [mempool_item1]
+
+
+@pytest.mark.asyncio
+async def test_new_peak_remove_from_pool() -> None:
+    mempool_manager = await instantiate_mempool_manager(get_coin_record_for_test_coins)
+    # Create a bunch of mempool items that spend TEST_COIN in order to setup
+    # a mapping of that coin's ID to several spend bundles
+    sb1_conditions = [[ConditionOpcode.CREATE_COIN, IDENTITY_PUZZLE_HASH, 1]]
+    sb1 = spend_bundle_from_conditions(sb1_conditions)
+    sb2_conditions = [[ConditionOpcode.CREATE_COIN, IDENTITY_PUZZLE_HASH, 2]]
+    sb2 = spend_bundle_from_conditions(sb2_conditions, TEST_COIN2)
+    sb3_conditions = [[ConditionOpcode.CREATE_COIN, IDENTITY_PUZZLE_HASH, 3]]
```

Comment:
it seems like a lot of this could be simplified by storing them in lists and create them with list-comprehension

Reply:
Done.